README.md

```diff
@@ -1 +0,0 @@
-# azure-storage-samples-for-python
```

Comment:
is this README.md under a folder?

Reply:
it is in the folder of the function, where we talked about moving it to. the other README that was outside of the function folder I deleted

Reply:
Oh i see, this readme is the one that was outside of the function folder that we deleted. The actual README that is in the function folder is still there